.github/workflows/release.yml

```diff
@@ -7,6 +7,11 @@ on:
 permissions:
   contents: write
 
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
We'll need to add these keys to the project. After the PR has been approved, I'll ask to have admin priviledges on this repo to add the sensitive information into the secrets.

Reply:
@luis02lopez @praveenperera , any follow up on that?

Reply:
@luis02lopez you can use

```
          role-to-assume: arn:aws:iam::529724559987:role/github-cdk-ci-ecr-role-repo-lava
```